log-server/src/main.rs

```diff
@@ -21,12 +22,14 @@ struct Data {
     result: Vec<ResultItem>,
 }
 
+#[allow(dead_code)]
```

Comment:
Why did you introduce this if you tackled the clippy warning we were getting?

Reply:
Because we do require this to parse the payload but we never actually use it. So, having it as a dead underling is somewhat justified.

Reply:
Can you point me where do you parse the payload?
To me it seems completely unused